external/fv3fit/tests/training/test_train.py

```diff
@@ -27,18 +27,32 @@
     "dense",
     "transformed",
 ]
+
+# unlabeled models that can be tested on all model configuration tests, but not those
+# that actually depend on the quality of supervised learning that occurs
+UNLABELED_TRAINING_TYPES = [
```

Comment:
Classification problems are "labeled" but will also probably fail some of the tests. Is there a name for this list that will also included classifiers? (e.g. NON_REGRESSION_TYPES)